StockAnalyzerService/Model/StockCandlestickViewModel.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace StockAnalyzerService.Model {
+
+    public class StockCandlestickViewModel : StockMetadata
```

Comment:
If this is the class that is used to pass into the post, it needs a property for Ticker and CompanyName in addition to the list of candlesticks